app/models/forem_instance.rb

```diff
@@ -28,7 +28,11 @@ def self.smtp_enabled?
     (Settings::SMTP.user_name.present? && Settings::SMTP.password.present?) || ENV["SENDGRID_API_KEY"].present?
   end
 
-  def self.private?
+  def self.invitation_only?
     Settings::Authentication.invite_only_mode?
   end
```

Comment:
Added this since there may be places in the codebase where we might actually want/need to check whether the particular forem instance is invite only or not. This will enable us to do so in a way that is consistent with the way we currently check these things throughout the codebase.